books/forms.py

```diff
@@ -105,3 +106,13 @@ class Meta:
         model = Book
         fields = ("status",)
         widgets = {"status": forms.HiddenInput()}
+
+    def save(self, request):
+        book = self.instance
+        if book.status == "AV":
+            BookHolder.objects.create(
+                book=book, holder=request.user, date_requested=timezone.now()
+            )
+            book.status = "RQ"
+        book = super().save(request)
+        return book
```

Comment:
It was meant as a conditional, when it's anything different from 'AV' it will just keep that status.. but I should probably separate  each action? I will do it at that point then I think.